src/frequenz/sdk/timeseries/grid.py

```diff
@@ -6,32 +6,135 @@
 This module provides the `Grid` type, which represents a grid connection point
 in a microgrid.
 """
+from __future__ import annotations
 
 import logging
+import uuid
 from collections.abc import Iterable
 from dataclasses import dataclass
+from typing import TYPE_CHECKING
 
-from ..microgrid.component import Component
+from frequenz.channels import Sender
+
+from ..microgrid.component import Component, ComponentMetadata
 from ..microgrid.component._component import ComponentCategory
 from ..microgrid.fuse import Fuse
```

Comment:
@frequenz-floss/python-sdk-team `Fuse` is defined in `frequenz.sdk.microgrid` while `Grid` is defined in `frequenz.sdk.timeseries`. Is this correct? just checking ...

Reply:
Probably not, nicer to keep them together in the timeseries package.

Reply:
It will be addressed in https://github.com/frequenz-floss/frequenz-sdk-python/issues/771